cmake/helper_functions.cmake

```diff
@@ -0,0 +1,70 @@
+# ign_generate_conf()
+# Generate conf/*.yaml files for projects
+macro(ign_generate_conf)
```

Comment:
Meybe change the name of this macro?

Reply:
How about ign_generate_conf_files, or could you suggest a better name?

Reply:
I have changed the name to configuration_functions as discussed.

Reply:
Let's keeping the same name as we couldn't come up with a better name